pkg/pillar/dpcreconciler/linuxitems/route.go

```diff
@@ -150,6 +150,9 @@ func (r Route) Dependencies() (deps []depgraph.Dependency) {
 				if len(addrs.IPAddrs) == 0 {
 					return false
 				}
+				if r.Gw.IsLinkLocalUnicast() {
```

Comment:
Why?

Reply:
It is explained in the commit message: https://github.com/lf-edge/eve/pull/5076/commits/3784d6301329f78ad9c862d8be6efb48c61cf6c4